lib/pymedphys/__init__.py

```diff
@@ -6,6 +6,8 @@
 from ._data import data_path, zenodo_data_paths, zip_data_paths
 from ._delivery import Delivery
 from ._gamma.implementation.shell import gamma_shell as gamma
+
+# from ._interp.interp import interp3d, multilinear_interp
```

Comment:
Uncommenting this line breaks the "clean test"(see [here](https://github.com/pymedphys/pymedphys/actions/runs/9660339581/job/26645696467#step:14:12)). It would be nice to have the underlying functions more easily runnable by users, but unsure how to facilitate / address that error. Could use some help!

Reply:
I suspect this should fix your issue:

https://github.com/pymedphys/pymedphys/pull/1867/files/75b76f880fc3f47ee862010a5d560cb3ec921361#r1652509093